snow/engine/snowman/config.go

```diff
@@ -16,7 +16,6 @@ import (
 // Config wraps all the parameters needed for a snowman engine
 type Config struct {
 	common.AllGetsServer
-	BlockTraversal      snow.BlockTraversal
```

Comment:
We don't need this to be part of the config anymore because we can just use `Consensus`